doc/administrator/web/index.md

```diff
@@ -45,11 +45,16 @@ Servlet Container Configuration
 -------------------------------
 Two environment variables needs to be set in your Servlet container for IRIDA to function correctly: `spring.profiles.active=prod` and `irida.db.profile=prod`.
 
-You can adjust these variables in Tomcat by editing (depending on your distribution) `/etc/tomcat/tomcat.conf` (CentOS) or `/etc/default/tomcat7` (Ubuntu), and finding the `JAVA_OPTS` variable and setting the variables as shown below:
+You can adjust these variables in Tomcat by editing (depending on your distribution) `/etc/tomcat/tomcat.conf` (CentOS) or creating a `setenv.sh` within `/opt/tomcat/bin` (Ubuntu). For CentOS you would find the `JAVA_OPTS` variable and for `Ubuntu` you would add the variables as shown below:
 
 ```
 JAVA_OPTS="-Dspring.profiles.active=prod -Dirida.db.profile=prod"
```

Comment:
Don't you also need to set JAVA_HOME? To make sure it uses java17?

Reply:
That is correct, I had it set in my .bashrc file that's why it was working for me without setting it. Updated in 7a5604de46298ba0f08714e23371877425ad1fb2